src/main/java/com/lms/learning_management_system/entities/service/UserService.java

```diff
@@ -0,0 +1,49 @@
+package com.lms.learning_management_system.entities.service;
+
+import com.lms.learning_management_system.entities.UserEntity;
+import com.lms.learning_management_system.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class UserService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    public UserEntity createUser(UserEntity user) {
+        return userRepository.save(user);
+    }
```

Comment:
We need to check if such a user exists. Its good practice to use request and response Dto